### ISSUE TITLE
Add option to disable --no-mtime flag

### ISSUE THREAD

**Issue by Maxstupo** [Maxstupo/ydl-ui#87]
> 
> For the time being I will have to use version [2.7.0](https://github.com/Maxstupo/ydl-ui/releases/tag/v2.7.0) since version [2.8.0](https://github.com/Maxstupo/ydl-ui/releases/tag/v2.8.0) does not keep the original date and time set by the server. Is there an option to revert this behaviour? If there is no such option I would like to suggest that you add a new check box, perhaps in **Download** tab in **Settings**:
> 
> - Keep the file date and time as set by the server
> 
> Or something like that. For me keeping the original date and time of a file is essential and important. Please consider adding such an option as a new check-box (it may be disabled by default).
> 

_Originally posted by @smaragdus in https://github.com/Maxstupo/ydl-ui/issues/83#issuecomment-856376797_

**Comment by smaragdus** [comment]
@Maxstupo 

Thank you very much for accepting my request! I didn't open a new issue myself because I thought that my suggestion was closely related to issue #83.

**Comment by smaragdus** [comment]
@Maxstupo 

Many thanks for adding this new check-box:

- Keep original date and time

I have just downloaded and tested version [2.8.1](https://github.com/Maxstupo/ydl-ui/releases/tag/v2.8.1) and the new option works as expected preserving the original date and time of the videos.

![YDL-UI 2 8 1 - 2021-06-20 - 002](https://user-images.githubusercontent.com/9025757/122659092-9623a380-d17c-11eb-80d3-569b6425ff57.png)

This option is very useful for me.

When I discovered your program in the autumn of 2018 (the first version I used I believe was [1.4.5](https://github.com/Maxstupo/ydl-ui/releases/tag/v1.4.5)) I didn't expect the development and maintenance to be so persistent. Thank you very much for all your work on this project!

Best regards
